Bixal/move.mil
# Feature/icon passer

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes
Refactored the icon attribute of the Json object in the weightCalcJsonController to use base urls. 

## Testing
1. Pull changes.
2. Install Module 
3. Browse to "http://move.mil.localhost:8000/parser/weight_calculator"
4. Check if icon is of the form: {BASE_URL} '\/themes\/custom\/move_mil\/assets\/img\/icons\/icons\/'{ICON}


## Screenshots

_Attach relevant before and after screenshots here._
